webapp/channels/src/components/admin_console/admin_definition.tsx

```diff
@@ -6934,7 +6934,7 @@ const AdminDefinition: AdminDefinitionType = {
                                     </ExternalLink>
                                 ),
                             },
-                            isHidden: true || it.not(it.licensedForFeature('SAML')),
+                            isHidden: it.not(it.licensedForFeature('SAML')),
```

Comment:
This is changing the previous existing behavior, that was intentional to disable this section from the admin console. Unless we have a good reason to renable it I wouldn't do this change.

Reply:
I see if it was intentionally hidden then i will remove the complete field it if required in future we can always get it back from the commit history